usagov/usagov-2021
# USAGOV-1730-wizard-manager-dragndrop-confirmation: Enabled confirmation dialogue in hierarchy_manager.hm_display_profile.wizard_manager.yml

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1730

## Description

Set the confirmation configuration for the hierarchy manager display profile to true.

## Type of Changes

- [ X ] New Feature
  - Add screenshot showing what it should look like
- [ X ] Drupal Config (requires "drush cim")

## Testing Instructions


1. As a content manager, navigate to /admin/structure/taxonomy/manage/wizard/overview.
2. Drag and drop a term in the tree.
3. A confirmation popup appears asking you to confirm your action.

### Change Requirements



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
5. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
